package.json

```diff
@@ -13,13 +13,15 @@
         "@typescript-eslint/eslint-plugin": "^2.28.0",
         "@typescript-eslint/parser": "^2.28.0",
         "eslint": "^6.8.0",
-        "eslint-config-airbnb-base": "^14.1.0",
         "eslint-config-prettier": "^6.10.1",
         "eslint-plugin-prettier": "^3.1.3",
-        "prettier": "^2.0.4"
+        "prettier": "2.0.4"
     },
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "check": "npx eslint --print-config src/* | npx eslint-config-prettier-check",
+        "lint": "npx eslint . --ext .js,.ts && echo 'Lint complete'",
+        "fix": "npx eslint . --fix . --ext .js,.ts && 'Fixed errors'"
```

Comment:
I have seen this as "lint-fix" elsewhere. Do we know if one or the other is more common? (Indeed's skeleton doesn't seem to have either one.)

Reply:
Also, I just realized that running the fix task gives errors, because it is missing the word 'echo'.